sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -252,3 +253,18 @@ def test_mutual_info_regression_X_int_dtype(global_random_seed):
     expected = mutual_info_regression(X_float, y, random_state=global_random_seed)
     result = mutual_info_regression(X, y, random_state=global_random_seed)
     assert_allclose(result, expected)
+
+
+@pytest.mark.parametrize(
```

Comment:
@netomenoci I push a piece of code that show how to make the parallelization if you are interested in.

Reply:
awesome, thanks :)

Reply:
Hi bary Wery good nice good luck bary